docker-entrypoint.sh

```diff
@@ -28,6 +35,7 @@ BASE_URL = '$BASE_URL'
 SECRET_KEY = '$(< /dev/urandom tr -dc A-Za-z0-9_\!\@\#\$\%\^\&\*\(\)-+= | head -c 32)'
 OAUTH2_CLIENT_ID = '$CLIENT_ID'
 OAUTH2_CLIENT_SECRET = '$CLIENT_SECRET'
+PLUGINS = ${PLUGS}
```

Comment:
No problem. I wouldn't normally just do stuff in python in a bash script but I think it makes sense to do it this way here because we're using python to generate a string representation of a python variable. Thanks.